stripe/src/main/java/com/stripe/android/PaymentController.java

```diff
@@ -400,9 +412,10 @@ public void onError(@NonNull Exception e) {
 
         private void startFrictionlessFlow() {
             mPaymentRelayStarter.start(new PaymentRelayStarter.Data(mPaymentIntent));
-            final ProgressDialog progressDialog = mProgressDialog.get();
-            if (progressDialog != null) {
-                progressDialog.dismiss();
+            final Activity activity = mActivityRef.get();
+            if (activity != null) {
+                LocalBroadcastManager.getInstance(activity)
```

Comment:
do we need this for challenge flow as well?

Reply:
The ChallengeActivity sends this broadcast for the challenge flow.

Should we move this to the `PaymentRelayActivity` to make it similar?